### PR TITLE
fixed broken link bug/SUPP-1268

### PR DESCRIPTION
**Pull request by GK230**

### CODE REVIEW

**Review by dannitahdruk — COMMENTED** [review]
better this need to be a parameter in the environment file and not hardcoded.

**Comment by kymmeh** [comment]
@GK230 please could you add the ticket number to the PR title?

File: .env [hunk 1]
```diff
@@ -4,3 +4,4 @@ NODE_ENV=production
 REACT_APP_HOTJAR_CODE=2685642
 REACT_APP_HOTJAR_CODE_VERSION=6
 REACT_APP_METADATA_CATALOG=https://modelcatalogue.cs.ox.ac.uk/hdruk_live
+NEWSLETTER_EXAMPLE=https://hdruk-ac-uk-1.hubspotpagebuilder.com/hdr-innovation-gateway-november-update
```
Comment (kymmeh) [inline .env:7]: Could you add URL to the name? ie `NEWSLETTER_EXAMPLE_URL`

File: src/pages/registration/CompleteRegistration.js [hunk 2]
```diff
@@ -925,7 +925,7 @@ const YourAccountForm = props => {
                             <a
                                 target='_blank'
                                 rel='noopener noreferrer'
-                                href='https://www.hdruk.ac.uk/infrastructure/gateway/privacy-policy/'>
+                                href={process.env.NEWSLETTER_EXAMPLE}>
```
Comment (kymmeh) [inline src/pages/registration/CompleteRegistration.js:928]: Once I've raised a PR I tend to click the "Files changed" tab and have a scan down (it's not bullet proof), but sometimes means you pick up things like this ^^ (don't think we want the privacy policy pointing to the newsletter)

**Review by kymmeh — CHANGES_REQUESTED** [review]

**Review by dannitahdruk — COMMENTED** [review]
- all hardcoded values who are in your path need to be moved in env file;
- since this repository is public, this env file must not be in Github (public), but there must be a dedicated page in Confluence for this.

**Comment by kymmeh** [comment]
> * all hardcoded values who are in your path need to be moved in env file;
> * since this repository is public, this env file must not be in Github (public), but there must be a dedicated page in Confluence for this.

@dannitahdruk we're aware of this and there's a task to remove .env from the repo. At the moment there is nothing sensitive within there.

**Comment by kymmeh** [comment]
@GK230 Please document any .env variables in the README.md (you'll see there's a section at the bottom)

**Review by kymmeh — APPROVED** [review]

**Review by loki-sinclair-hdruk — APPROVED** [review]

**Pull request merged**